extra/failover-testing/docker-compose.failover-server.yml

```diff
@@ -162,7 +162,7 @@ services:
             - CONFIG_PROP=config.properties
 
     mender-elasticsearch-2:
-        image: mendersoftware/elasticsearch:2.4
+        image: mendersoftware/elasticsearch:2.2.2
```

Comment:
@mchalski I guess 2.2.2 tag is missing in docker hub https://hub.docker.com/r/mendersoftware/elasticsearch/tags/
We need to push it there or use elasticsearch:2.2.2 instead of mendersoftware/elasticsearch

Reply:
ok, fixed to just `elasticsearch:2.2.2`.

@maciejmrowiec is this what you were referring to? or do you mean 'unsupportable' in a broader sense, as in this whole PR is invalid?

Reply:
Meant in general, unless you want to contribute 2.4->2.2 downgrade support migration to ES ;)